src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,9 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+
+    // Drive train related
+    public static final double TRACK_WIDTH_METERS = 0.68;
```

Comment:
We have this in Drive Constants https://github.com/PearceRobotics/rapid-react-2022/blob/main/src/main/java/frc/robot/constants/DriveConstants.java#L10

Can we use that variable instead?